### PR TITLE
Update page title

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ nav:
           - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
           - Performance Analysis: observability/analyze-performance.md
     - Insights:
-          - Derive Insights: insights/derive-insights.md
+          - View API Insights: insights/derive-insights.md
```
Comment (Nashaath) [inline en/mkdocs.yml:73]: Would be good to change the file name too since we generally follow that practice.

**Review by Nashaath — CHANGES_REQUESTED** [review]
Added a minor suggestion. There are a couple of minor typos in the "View API Insights" page that need to be fixed too.

File: en/mkdocs.yml (hunk 1 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Corrected

File: en/docs/insights/derive-insights.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Derive Insights
+# View API Insights
```
Comment (Nashaath) [inline en/docs/insights/derive-insights.md:1]: Since we don't need this file shall we remove.

**Review by shaniR — APPROVED** [review]

**Review by Nashaath — APPROVED** [review]

**Pull request merged**